contracts/strategies/allocation/direct-grants-committee/README.md

```diff
@@ -0,0 +1,91 @@
+Spec: Direct Grants Pool with Committee Voting
+---------------------------------
+
+## Overview 
+Many web3 projects and traditional orgs operate a direct grants program, where they set aside funds and enable small committees to decide how to distribute those funds for a given purpose. Applications are solicited for projects that support the pool's goal.
+
+## Component Quickview
+As laid out in the [components overview](https://docs.google.com/document/d/1qoOP07oMKzUCyfb4HbnyeD6ZYEQa004i5Zwqoy7-Ox8/edit), each allocation strategy consists of some key components. This is a quick overview of the relevant components for this strategy.
+- **Recipient eligibility**: must hold credential and must include proposal
+- **Acceptance method**: programmatic
+- **Voter eligibility**: allowlist
+- **Voting method**: one-voter-one-vote
+- **Calculation method**: simple threshold
+- **Allocation shape**: discrete amount
+
+## Spec
+### Custom Variables
+This strategy will need the following custom variables:
+- `poolOpen` - boolean value that represents whether the pool is accepting new applications
+- `poolMetadata` - off-chain metadata about the pool
+- `committee` - allowlist of eligible voters for the pool
+- `acceptanceThreshold` - number of eligible voters who must vote yes on a proposal to unlock the funds. Number must be >=1 and <=the total number of committee voters. 
+- `eligibilityVerification` - the list of on-chain verifications that must be present on the applicant's identity address to be eligible
+- `proposal` — contains the application's proposal for grant funding
```

Comment:
Is this going to be just a pointer to the proposal or bytes for a transaction/function call?